plugin-build/src/main/kotlin/io/sentry/android/gradle/extensions/TracingInstrumentationExtension.kt

```diff
@@ -77,5 +89,12 @@ enum class InstrumentationFeature {
      * This feature uses bytecode manipulation and adds an OnDestinationChangedListener to all
      * navigation controllers used in Jetpack Compose.
      */
-    COMPOSE
+    COMPOSE,
+
+    /**
+     * When enabled the SDK will create breadcrumbs when using Logcat
+     * This feature uses bytecode manipulation and replaces Logcat
+     * with Sentry-specific implementations.
+     */
+    LOGCAT
```

Comment:
I don't think we need this as we already have `logcat.enabled` now right?

Reply:
yeah I was about to comment the same, we probably don't need it